tests/data/mini_c_test/b_test.cc

```diff
@@ -17,42 +17,54 @@ TEST(b_test, TestSignalChain_1)
 
 TEST(b_test, TestSignalChain_2)
 {
-  CREATE_MOCK(mymock);
+  LOCAL_MOCK(mymock);
   EXPECT_CALL(mymock, a_get_y2()).WillRepeatedly(Return(13));
   EXPECT_CALL(mymock, c_set_u2(13));
   b_step();
 }
 
 TEST(b_test, TestSignalChain_3)
 {
-  CREATE_MOCK(mymock);
+  LOCAL_MOCK(mymock);
   EXPECT_CALL(mymock, a_get_y3_and_set_u5(_)).WillRepeatedly(Return(13));
   EXPECT_CALL(mymock, c_set_u3_and_u4(13, _));
   b_step();
 }
 
-TEST(b_test, TestSignalChain_4)
+/* Introducing a test fixture */
+
+class B : public Test {
+  void SetUp() override {
+     mock = CREATE_MOCK();   // Create the mock handle in the fixture
+     // Now some generic reactions can be set up
+     ON_CALL(*mock, a_get_y5).WillByDefault(Return(13));
+  }
+  void TearDown() override {
+     DESTROY_MOCK();  // Tear down the mock handle to finalize its expecations

```

Comment:
Wouldn't this be correct and more readable?
```suggestion
     DESTROY_MOCK(mock);  // Tear down the mock handle to finalize its expecations

```

Reply:
Dunno. `CREATE_PERSISTENT_MOCK()` has no argument, and so why should `DESTROY_` have. It doesn't need it.
And I didn't want to give `CREATE_` an argument, since the difference would be to just put the assignment into the macro, which might be hindering usecases where this is not exactly what you wanted to do.

`CREATE_PERSISTENT_MOCK` with argument also would look like `CREATE_MOCK()`, which brings its argument into existence.

Reply:
Dunno. `CREATE_PERSISTENT_MOCK()` has no argument, and so why should `DESTROY_` have. It doesn't need it.
And I didn't want to give `CREATE_` an argument, since the difference would be to just put the assignment into the macro, which might be hindering usecases where this is not exactly what you wanted to do.

`CREATE_PERSISTENT_MOCK` with argument also would look like `CREATE_MOCK()`, which brings its argument into existence.

Reply:
Yes, you are right. Forget those arguments. Just rename the macros as proposed to be compatible to the latest release.